frontend2/src/components/tables/submissions/SubHistoryTable.tsx

```diff
@@ -26,12 +30,19 @@ interface SubHistoryTableProps {
   handlePage: (page: number) => void;
 }
 
+
+
+// TODO: should i pass episodeId down as a prop?
```

Comment:
No need to pass `episodeId` down, you can access it anywhere in the App tree using the `useEpisodeId` hook! :sunglasses: